extension/fts/src/function/query_fts_index.cpp

```diff
@@ -289,12 +289,13 @@ void QFTSVertexCompute::vertexCompute(const graph::VertexScanState::Chunk& chunk
 }
 
 void QueryFTSAlgorithm::exec(processor::ExecutionContext* executionContext) {
+    executionContext->clientContext->setUseInternalTable(true /* useInternalTable */);
```

Comment:
I don't follow where we set use internal to false.

Reply:
After the execution of the execFunction, we set it to false.